src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/reference/inputs/EceWorkExperienceReferenceDetails.vue

```diff
@@ -0,0 +1,229 @@
+<template>
+  <v-row>
+    <v-col cols="12" md="12" lg="12" xl="12">
+      <h3>Work experience hours</h3>
+      <div role="doc-subtitle">Please provide as much detail as possible.</div>
+      <v-row class="mt-5">
+        <v-col cols="12" md="8" lg="6" xl="4">
+          Applicant indicated you observed them work:
+          <b>50 hours</b>
+        </v-col>
+      </v-row>
+      <v-row>
+        <v-col cols="12" md="8" lg="6" xl="4">
+          <v-text-field
+            type="number"
+            :rules="[Rules.required('Enter a whole number greater than zero (0)')]"
+            label="Total number of hours you observed the applicant working"
+            variant="outlined"
+            color="primary"
+            maxlength="10"
+            hide-details="auto"
+            @input="updateField('hours', $event)"
+          />
```

Comment:
I suggest using v-text-field + isNumber on keypress.

Reason being using the type = "number" will allow the characters "e" and "." to be added to the field.